### PR TITLE
dev release 708

### PR DESCRIPTION
**Pull request by Turtus**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## BUILD AND DEPLOY
```
Comment (hamedtayeban) [inline readme.md:2]: `bc1qhva87e9lc2lhnyvjptj05szqvjw886jvlmx5rt`
[]()https://blockchair.com/bitcoin/address/bc1qhva87e9lc2lhnyvjptj05szqvjw886jvlmx5rt?from=trustee

Comment (hamedtayeban) [reply readme.md:2]: []()bc1qhva87e9lc2lhnyvjptj05szqvjw886jvlmx5rt
https://blockchair.com/bitcoin/address/bc1qhva87e9lc2lhnyvjptj05szqvjw886jvlmx5rt?from=trustee

Comment (hamedtayeban) [reply readme.md:2]: bc1qhva87e9lc2lhnyvjptj05szqvjw886jvlmx5rt
https://blockchair.com/bitcoin/address/bc1qhva87e9lc2lhnyvjptj05szqvjw886jvlmx5rt?from=trustee

Comment (hamedtayeban) [inline readme.md:2]: bc1qhva87e9lc2lhnyvjptj05szqvjw886jvlmx5rt
https://blockchair.com/bitcoin/address/bc1qhva87e9lc2lhnyvjptj05szqvjw886jvlmx5rt?from=trustee

File: readme.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
 
 
+
```
Comment (hamedtayeban) [inline readme.md:4]: bc1qhva87e9lc2lhnyvjptj05szqvjw886jvlmx5rt
https://blockchair.com/bitcoin/address/bc1qhva87e9lc2lhnyvjptj05szqvjw886jvlmx5rt?from=trustee

Comment (hamedtayeban) [reply readme.md:4]: bc1qhva87e9lc2lhnyvjptj05szqvjw886jvlmx5rt
https://blockchair.com/bitcoin/address/bc1qhva87e9lc2lhnyvjptj05szqvjw886jvlmx5rt?from=trustee

**Review by hamedtayeban — APPROVED** [review]
```
bc1qhva87e9lc2lhnyvjptj05szqvjw886jvlmx5rt

```

**Comment by hamedtayeban** [comment]
[]()https://blockchair.com/bitcoin/address/bc1qhva87e9lc2lhnyvjptj05szqvjw886jvlmx5rt?from=trustee

**Comment by hamedtayeban** [comment]
#708

File: readme.md (hunk 1 above)
Comment (Turtus) [reply readme.md:2]: spam(

**Pull request merged**